mobile-first/index.html

```diff
@@ -0,0 +1,94 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  
+  <title>Great Idea - Responsive I</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+<body class="container">
+<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+
+  <nav class="navigation">
+    <a href="services.html">Services</a>
+    <a href="">Product</a>
+    <a href="">Vision</a>
+    <a href="">Features</a>
+    <a href="">About</a>
+    <a href="">Contact</a>
+
+    <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+  </nav>
+
+  <header>
+    <div class="main-heading">
+      <h1>Innovation
+        <br> On
+        <br> Demand
+      </h1>
+      <button class="get-started">
+            Get Started
+      </button>
+    </div>
+    <img class="header-image" src="img/header-img.png" alt="Image of a code snippet.">
+
+  </header>
+
+  <div class="first-section">
+    <section class="features-about">
+      <h3>Features</h3>
```

Comment:
Try to use more general/descriptive class names. If you changed the title of one of the sections below, or added an additional section, this name wouldn't make sense anymore. Maybe `info-block` or `text-descriptions`